source/main/modules/window-manager/index.ts

```diff
@@ -51,32 +54,35 @@ import { WindowPosition } from './types.d'
 import askFileDialog from './dialog/ask-file'
 // import dragIcon from '../../assets/dragicon.png'
 
+// issue : Add tray to MacOS #4
+let tray: Tray | null = null
+
 interface QuicklookRecord {
   path: string
   win: BrowserWindow
 }
 
 export default class WindowManager {
-  private _mainWindow: BrowserWindow|null
+  private _mainWindow: BrowserWindow | null
   private readonly _qlWindows: QuicklookRecord[]
-  private _printWindow: BrowserWindow|null
-  private _logWindow: BrowserWindow|null
-  private _statsWindow: BrowserWindow|null
-  private _defaultsWindow: BrowserWindow|null
-  private _preferences: BrowserWindow|null
-  private _customCSS: BrowserWindow|null
-  private _aboutWindow: BrowserWindow|null
-  private _tagManager: BrowserWindow|null
-  private _pasteImageModal: BrowserWindow|null
-  private _errorModal: BrowserWindow|null
-  private _printWindowFile: string|undefined
+  private _printWindow: BrowserWindow | null
+  private _logWindow: BrowserWindow | null
+  private _statsWindow: BrowserWindow | null
+  private _defaultsWindow: BrowserWindow | null
+  private _preferences: BrowserWindow | null
+  private _customCSS: BrowserWindow | null
+  private _aboutWindow: BrowserWindow | null
+  private _tagManager: BrowserWindow | null
+  private _pasteImageModal: BrowserWindow | null
+  private _errorModal: BrowserWindow | null
+  private _printWindowFile: string | undefined
```

Comment:
Don't make whitespace changes to code unrelated to the pull request. I don't think the Zettlr developers will approve the whitespace changes.

Reply:
I will fix it and make new commit